src/tensor_ops/conv.rs

```diff
@@ -0,0 +1,302 @@
+use crate::prelude::*;
+
+pub fn conv2d<
```

Comment:
Adding a `const CFG: ConvConfig` could be really helpful to reduce const generic parameters here. Will require an additional feature to enable usage of structs in const

Reply:
this could be really helpful, especially if the amount of filters can be set independently, so there could be a default convconfig for a kernel size of 3x3 with 0 padding etc. And conv2d is nightly anyway, so another feature wouldn't hurt much

Reply:
a default option like this could use typealiasing though and would not require extra-features